js/transform.js

```diff
@@ -222,6 +232,15 @@ function displayForm(e) {
 }
 
 function generateAndDisplayForm(formId, dest) {
+    const INTEGER_AFTER_UNDERSCORE_REGEX = /(.*)_([0-9]+)/;
+    const match = formId.match(INTEGER_AFTER_UNDERSCORE_REGEX);
+    if (match) {
+        var formName = match[1]
+        var formIterator = match[2]
+    } else {
+        throw Error('Could not parse formId with regex. Expected formId to be of form .*_([0-9]+). Instead formId was ' + formId);
```

Comment:
[Dust] A common linting convention is to use a template string `Could not parse formId... ${formId}` instead of concatenation. I think it's mainly for readability

https://stackoverflow.com/questions/27565056/es6-template-literals-vs-concatenated-strings
https://eslint.org/docs/latest/rules/prefer-template